jvm-packages/xgboost4j/src/main/java/ml/dmlc/xgboost4j/java/Booster.java

```diff
@@ -432,6 +432,7 @@ public void setVersion(int version) {
    */
   public byte[] toByteArray() throws XGBoostError {
     byte[][] bytes = new byte[1][];
+    checkHandle();
```

Comment:
let's make jvm crash, I just do not want to downgrade the issue which has a root cause in native layer

Reply:
But SIGSEGV is super-hard to debug, right? I would suggest we throw some meaningful exception when calling API on an reclaimed Booseter.

I can make another PR to throw the exception at native level. Do you think that makes sense?

Reply:
well....the issue is that you can identify a booster is reclaimed, because we set handle to 0 explicitly in java code. So using this approach is not a real (comprehensive) fix

if you really want to tackle this problem, we need to find a way to justify a reclaimed booster without the assistance in our java code

Reply:
nvm, it's fine to use `handle = 0` to identify a reclaimed booster, can you do it in another PR?

Reply:
Sorry, I don't get what do you mean by `justify a reclaimed booster`?

Reply:
Sure. Sounds good~